addon/templates/components/polaris-connected/item.hbs

```diff
@@ -1 +1,9 @@
-{{yield}}
+{{!-- template-lint-disable no-invalid-interactive --}}
+<div
+  class={{this.classes}}
+  ...attributes
+  {{on "focus" this.handleFocus}}
+  {{on "blur" this.handleBlur}}
```

Comment:
Do we need to prevent bubbling on these events? 🤔

Reply:
I don't think anything changes from what we had before with this transition
but we'll definitely find out once we start testing